code/modules/jobs/job_types/security_officer.dm

```diff
@@ -142,6 +142,9 @@ GLOBAL_LIST_EMPTY(security_officer_distribution)
 			if (pda.owner in partners)
 				targets += "[pda.owner] ([pda.ownjob])"
 
+	if (!targets.len)
+		continue
```

Comment:
code/modules/jobs/job_types/security_officer.dm:146:error: continue: not in a loop

Reply:
well that makes sense, i tacked that part on after realizing 😛

Reply:
```suggestion
	if (!targets.len)
		return
```